### ISSUE TITLE
Unsolved import does not go away after adding Import missing file.

### ISSUE THREAD

**Issue by zhouwangyang** [microsoft/PTVS#5398]
repro step:
1.Create python application in VS
2. Tying code in default .py file
import AbsentMoudule
3.Make sure unsolved import message in error list window
4.Right-click python application and select add->Existing item......
5.Select AbsentMoudule.py from another folder out of the python application workspace and then click button "Add" 
6.Make sure AbsentMoudule.py can be added under python application project

except :
unsolved import message in error list window should disappear and  the squiggles should go away after adding the missing file

actual:
the squiggles did not go away and the unsolved import message was still in the error list window.

testing environment:
![image](https://user-images.githubusercontent.com/39871640/60499914-f00ad480-9ceb-11e9-8d54-903f5a36c946.png)

detail steps:

![import missing](https://user-images.githubusercontent.com/39871640/60500050-3eb86e80-9cec-11e9-99a5-7e1c186d7f9d.gif)

**Comment by huguesv** [comment]
It seems to work ok if you open the file after adding it.

**Comment by zhouwangyang** [comment]
yes, if open the file after adding it, the squiggles will go away and the error in the error list will disappear. @huguesv
